earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -190,7 +139,8 @@ ()
 {
     // keep hold of it as a mutable so we can change/create it and
     // have the changes automatically saved
-    Mutable${name}AssetD asset = Find(ref_ $forwardtypearg);
+    using ::Find;
+    Mutable${name}AssetD asset = Find<Mutable${name}AssetD>(ref_, $typeref, ToString("$subtype"));
```

Comment:
This is very minor, but you can get the same effect by leaving out the `using` clause by putting the scope resolution operator on the beginning of the Find call:
```
Mutable${name}AssetD asset = ::Find<Mutable${name}AssetD>(ref_, $typeref, ToString("$subtype"));
``